### DISCUSSION TITLE
Shared Electron

### DISCUSSION THREAD

**Discussion by cbertelli** [minbrowser/min#2291] [Ideas]
I don't know if it happens to others, but my computer is bloated with several dozen applications based on Electron.

Would it be possible to use the latest Electron and share it? I understand it seems easy and it asks for some API stability, so it should be better to ask to Electron developers, but it's really so much space lost that I would like very much to save.

Just a symbolic link sometimes solves the problem, but doing this by design could be better.

**Comment by PalmerAL** [comment]
Yeah, a good solution for this would require the Electron project to implement something, it's not something an individual app can implement very well.

I think it would be possible to do now though. Historically, Electron wasn't very stable (neither forwards- nor backwards-compatible), and so it would have been very difficult to do this, but it's gotten a lot more stable recently, and it would probably work to have a single copy of the couple most recent major versions of Electron. I'm not sure how they'd get updated though.
